UPGRADE-4.0.md

```diff
@@ -184,6 +184,31 @@ Form
    }
    ```
 
+ * Using the "choices" option in ``CountryType``, ``CurrencyType``, ``LanguageType``,
+  ``LocaleType``, and ``TimezoneType`` is now ignored. Use the "choice_loader" option
+   instead or explicitly set it to ``null``.
```

Comment:
Yes I think so, if one wants to override choices, he should not be force to override the loader.

It makes no sense IMO to enforce lazy load something that is already loaded, if you have your choices, just pass them, do not encapsulate them in an objet that will eventually call a function to return the initial thing.